packages/blocks/src/_common/components/rich-text/rich-text-operations.ts

```diff
@@ -861,6 +862,18 @@ function handleParagraphBlockForwardDelete(
     // TODO
     return false;
   } else {
+    const ignoreForwardDeleteFlavourList: Flavour[] = [
+      'affine:database',
+      'affine:image',
+      'affine:code',
+      'affine:attachment',
+    ];
+    if (
+      nextSibling &&
+      ignoreForwardDeleteFlavourList.includes(nextSibling.flavour as Flavour)
+    )
+      return true;
```

Comment:
Please use this style:

```
if (someLongCondition) {
  return true;
}
```

Tried but unable to directly push to your branch to tweak it :)